packages/util/src/__specs__/__snapshots__/css.spec.ts.snap

```diff
@@ -0,0 +1,20 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`css() 1`] = `
```

Comment:
Snapshot looks a little odd.  Is this supposed to be prettified or minified? It seems to be somewhere in the middle.

Reply:
need to be deleted